src/nomad_simulations/schema_packages/numerical_settings.py

```diff
@@ -56,13 +61,29 @@ def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
 
 class Mesh(ArchiveSection):
     """
-    A base section used to specify the settings of a sampling mesh. It supports uniformly-spaced
-    meshes and symmetry-reduced representations.
+    A base section used to specify the settings of a sampling mesh.
+    It supports uniformly-spaced meshes and symmetry-reduced representations.
     """
 
+    type = Quantity(
```

Comment:
I'm not finding the section nor quantity... I checked, and I did branch of the most recent version of `develop`

Reply:
https://github.com/nomad-coe/nomad-simulations/blob/develop/src/nomad_simulations/schema_packages/model_system.py#L153

Reply:
Okay, I misread the file. Thx for the link.
I'm still unsure of where to specify the coordinates themselves, though:

`Mesh` refers to the `points` which are dependent on the coordinate system. `GeometricalSpace` only uses the coordinate system in the normalization.
The current implementation always assumes `x, y, z` coordinates: it does not automatically convert to `r, theta, phi` when changing to `spherical` for example...

We can likely retain the original distinction/semantics, but I suggest _dependency injection_ for any geometric functionality, meaning that we pass a `CoordinateSystem` class implementing a set of routines to the section initializer. Further normalization is then handled by the section calling those routines, as usual.
Meanwhile, we use _polymorphism_ to specialize `CoordinateSystem` into its various kinds. (Note that Cartesian should be clarified where it assumes 90 angles + we're missing ellipsoidal.)

This is going quite outside this PR. For now, I'll leave `Mesh` as was.
I can open a new issue, assign myself, and have you review it. This would be for after the holidays, though. Just give it a thought.